packages/arb-token-bridge-ui/src/components/common/Transition.tsx

```diff
@@ -1,53 +1,52 @@
-import React, { Fragment } from 'react'
-import { Transition as HeadlessUITransition } from '@headlessui/react'
+import { Transition as HeadlessUiTransition } from '@headlessui/react'
+import { PropsWithChildren } from 'react'
+import { twMerge } from 'tailwind-merge'
 
-type TransitionDuration = 'fast' | 'normal'
+type TransitionSpeed = 'slow' | 'normal' | 'fast'
 
-function getDurationNumbers(
-  duration: TransitionDuration
-): [enterDuration: number, leaveDuration: number] {
-  switch (duration) {
-    case 'fast':
-      return [150, 100]
-
-    case 'normal':
-      return [300, 200]
-  }
+type TransitionOptions = {
+  enterSpeed?: TransitionSpeed
+  leaveSpeed?: TransitionSpeed
+  unmountOnLeave?: boolean
```

Comment:
HeadlessUI transition doesn't work well with react-virtualized, especially Lists.

We need to set `unmountOnLeave` to false for this case, otherwise the exit transition doesn't work.

See `TokenButton.tsx` and `NetworkSelectionContainer.tsx`